nats.go

```diff
@@ -4446,7 +4446,7 @@ func (s *Subscription) ClearMaxPending() error {
 
 // Pending Limits
 const (
-	// DefaultSubPendingMsgsLimit will be 512k msgs.
+	// DefaultSubPendingMsgsLimit will be 524,288 msgs.
```

Comment:
Why not 512Kb?

Reply:
I guess I assumed using Kb as unit in a non-storage context was not common. Happy to change if you think otherwise.. but when I see a `b` I immediately associate storage rather than a discrete number.

Reply:
Added another commit.. like so?